### PR TITLE
added a link for the beacon deposit contract on etherscan

### PR DESCRIPTION
**Pull request by mcmoodoo**
## Summary by CodeRabbit

- **Documentation**
  - Updated documentation to specify that node operators must deposit ether into a specific smart contract on Ethereum when participating in block validation.

### CODE REVIEW

**Comment by mcmoodoo** [comment]
@coderabbitai review

File: public/content/developers/docs/consensus-mechanisms/pos/rewards-and-penalties/index.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: Learn about the in-protocol incentives in proof-of-stake Ethereum.
 lang: en
 ---
 
-Ethereum is secured using its native cryptocurrency, ether (ETH). Node operators that wish to participate in validating blocks and identifying the head of the chain deposit ether into a smart contract on Ethereum. They are then paid in ether to run validator software that checks the validity of new blocks received over the peer-to-peer network and apply the fork-choice algorithm to identify the head of the chain.
+Ethereum is secured using its native cryptocurrency, ether (ETH). Node operators that wish to participate in validating blocks and identifying the head of the chain, deposit ether into a [deposit smart contract](https://etherscan.io/address/0x00000000219ab540356cbb839cbe05303d7705fa) on Ethereum. They are then paid in ether to run validator software that checks the validity of new blocks received over the peer-to-peer network and apply the fork-choice algorithm to identify the head of the chain.
```
Comment (wackerow) [inline public/content/developers/docs/consensus-mechanisms/pos/rewards-and-penalties/index.md:7]: Hey @mcmoodoo, appreciate the thought here.

We have a dedicated page for the [deposit-contract](https://ethereum.org/en/staking/deposit-contract/) which has a link to Etherscan already. For the sake of security/user safety, we only declare the deposit contract address [in one place](https://github.com/ethereum/ethereum-org-website/blob/dev/src/data/addresses.ts) in the entire repo to have one canonical source of truth that we use, which we can easily spot if it's ever changed. 

Also, if we're going to link out and use the word "deposit" here, I would change it to reference "the deposit contract" (as there is only one, and it's named "deposit contract"... remove "smart")

```suggestion
Ethereum is secured using its native cryptocurrency, ether (ETH). Node operators that wish to participate in validating blocks and identifying the head of the chain, deposit ether into the [deposit contract](/staking/deposit-contract/) on Ethereum. They are then paid in ether to run validator software that checks the validity of new blocks received over the peer-to-peer network and apply the fork-choice algorithm to identify the head of the chain.
```

**Review by wackerow — CHANGES_REQUESTED** [review]

File: public/content/developers/docs/consensus-mechanisms/pos/rewards-and-penalties/index.md (hunk 1 above)
Comment (corwintines) [reply public/content/developers/docs/consensus-mechanisms/pos/rewards-and-penalties/index.md:7]: Agree with this suggestion here. Going to pull this in to merge.

**Review by corwintines — APPROVED** [review]
Thanks @mcmoodoo

**Pull request merged**